spartan/aztec-network/templates/prover-agent.yaml

```diff
@@ -17,9 +17,19 @@ spec:
         {{- include "aztec-network.selectorLabels" . | nindent 8 }}
         app: prover-agent
     spec:
-      {{- if .Values.proverAgent.nodeSelector }}
```

Comment:
If nodeselector is no longer used, can you purge it?

Reply:
This `nodeSelector` item is not part of the standard deployment files. It is only present in `./values/gcp-proving-test.yaml` (last used by @PhilWindle ). Will confirm in planning session today if that entire file can be purged - now that GCP prover deploys use diff configurations.

Reply:
Am I able to keep the ability to run GCP proving tests using non-spot instances?

Reply:
Yes, spot support is off by default with his PR. 

```
proverAgent:
  enabled: true
  replicas: 1
  gke:
    spotEnabled: false
```

Reply:
If `spotEnabled` is `false`, the pods default to non-spot nodes.